### PR TITLE
Expanding ban command feature set (unban and isbanned)

### PR DESCRIPTION
**Pull request by jonfriesen**
Built two additional commands (unban, isbanned) and made some changes to the existing ban command.

**Changes**
- ban now shows notification to admin user in bad
- custom message if admin tries to ban themselves

**Additions**
- unban command: /unban username
- isbanned command: /isbanned or /isbanned user (shows all banned users or if a specific user is banned)

I built this PR based on #1075 and it should be noted that the existing ban command already works on offline users.

### CODE REVIEW

File: JabbR/Hubs/Chat.cs [hunk 1]
```diff
@@ -1128,6 +1128,21 @@ void INotificationService.BroadcastMessage(ChatUser user, string messageText)
             }
         }
 
+        void INotificationService.IsBanned()
+        {
+            var users = _repository.Users.Where<ChatUser>(s => s.IsBanned == true).Select(s => s.Name).OrderBy(s => s);
```
Comment (davidfowl) [inline JabbR/Hubs/Chat.cs]: nit: Style:

```
var users = _repository.Users.Where(u => u.IsBanned)
                             .Select(u => u.Name)
                             .OrderBy(u => u.Name);
```

**Comment by davidfowl** [comment]
Overall comments:
- JS Hint failed (take a look at those errors)
- Change casing to Unban instead of UnBan

**Comment by jonfriesen** [comment]
Thanks for the advice @davidfowl, made the appropriate changes. I'm unsure of how to kick off a CI build.

**Comment by davidfowl** [comment]
You need to rebase your changes. You have some conflicts.

**Comment by jonfriesen** [comment]
I'm pretty sure I just screwed everything up. Tried to rebase, resolve conflicts and commit them.

**Comment by davidfowl** [comment]
The diff looks fine but you have 15 commits. Do you know how to squash them?

**Comment by jonfriesen** [comment]
I have to do an interactive rebase back 16 commits merging into the first one. Currently I have conflicts with 6 files from the original rebase. I'll have to sort those out before I can squash.

**Comment by jonfriesen** [comment]
@davidfowl After further investigation, I've messed up this PR pretty bad (my first attempt at contributing to OSS). Probably a good idea to kill it and I'll try again soon :( thanks for the help

**Pull request closed without merging**